src/apim_nodo_services.tf

```diff
@@ -174,3 +174,174 @@ resource "azurerm_api_management_api" "apim_nodo_pagamenti_api_ec_svr" {
 #   }
 
 # }
+
+############################
+## WS node for psp (NM3) ##
+############################
+locals {
+  apim_node_for_psp_api = {
+    display_name          = "Node for PSP WS (NM3)"
+    description           = "Web services to support PSP in payment activations, defined in nodeForPsp.wsdl"
+    path                  = "api/node-for-psp"
+    subscription_required = false
+    service_url           = null
+  }
+}
+
+resource "azurerm_api_management_api_version_set" "node_for_psp_api" {
+  name                = format("%s-node-for-psp-api", var.env_short)
+  resource_group_name = azurerm_resource_group.rg_api.name
+  api_management_name = module.apim.name
+  display_name        = local.apim_node_for_psp_api.display_name
+  versioning_scheme   = "Segment"
+}
+
+resource "azurerm_api_management_api" "apim_node_for_psp_api_v1" {
+  name                  = format("%s-node-for-psp-api", var.env_short)
+  api_management_name   = module.apim.name
+  resource_group_name   = azurerm_resource_group.rg_api.name
+  subscription_required = local.apim_node_for_psp_api.subscription_required
+  version_set_id        = azurerm_api_management_api_version_set.node_for_psp_api.id
+  version               = "v1"
```

Comment:
future needs , we should link this version to https://github.com/pagopa/pagopa-api version